docs/_articles/en/v3.x/salesforce-graph-engine/graph-engine-recommendations.md

```diff
@@ -0,0 +1,132 @@
+# Writing Graph Engine-Friendly Code
```

Comment:
This file is missing from the sidebar, and having a few rendering errors. I can fix those myself if you want, or I could walk you through fixing them.

Reply:
@jfeingold35 Added to sidebar. The file renders nicely in Preview, so I'm not sure what the other errors are?